xtras/src/supervisor.rs

```diff
@@ -249,8 +267,11 @@ struct GetMetrics;
 
 #[cfg(test)]
 mod tests {
+    use crate::SendAsyncSafe;
+
```

Comment:
:wrench: Unwanted newline.

Reply:
ah, that rust-analyzer again! I think it tries to establish the convention (not used here) that crate elements should come in its own block at the top. fixed :)